packages/rest/test/unit/coercion/paramStringToBoolean.unit.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {testCoercion} from './utils';
+
+describe('coerce param from string to boolean', () => {
+  it("value 'false' is coerced to false", async () => {
```

Comment:
`it('value "false" is coerced to false')` - please use the same style in other tests too.